bolt-sidecar/src/config/mod.rs

```diff
@@ -144,7 +143,7 @@ fn remove_empty_envs() -> eyre::Result<()> {
     for item in env::vars() {
         let (key, val) = item;
         if val.trim().is_empty() {
-            debug!("removing empty env var: {}", key);
+            println!("Removing empty environment variable with key: {}", key);
```

Comment:
if this has been tested and works, can you remove this? I would try to avoid printing to stdout if possible